src/main/java/org/springframework/samples/petclinic/vet/VetController.java

```diff
@@ -1,3 +1,7 @@
+// Line 1
+//Line 2
+//Line 3
```

Comment:
🧑‍🏫 says:

54: Changing the `PageRequest.of(page - 1, pageSize)` to `PageRequest.of(page, pageSize)` disrupts the pagination logic, as the page index should start from 0; this change will cause the first page of results to be skipped entirely.